components/raftstore/src/store/worker/region.rs

```diff
@@ -407,12 +405,21 @@ where
             coprocessor_host,
             router,
             pd_client,
-            pool: Builder::new(thd_name!("snap-generator"))
-                .max_thread_count(cfg.value().snap_generator_pool_size)
+            pool: YatpPoolBuilder::new(DefaultTicker::default())
+                .name_prefix("snap-gen")
```

Comment:
why change the thread name?

Reply:
It's for testing. I will change it back.